<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Interference Classification</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
      <item>
       <widget class="QGroupBox" name="path_box">
        <property name="title">
         <string>Path to Emit Project</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="file_path_box">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="file_select_btn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="design_name_box">
        <property name="title">
         <string>Design Name</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="design_name_dropdown">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="protection_tab">
       <attribute name="title">
        <string>Protection Level</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="protection_select_box">
            <property name="title">
             <string>Protection Level Thresholds</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="damage_check">
               <property name="text">
                <string>Damage</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="overload_check">
               <property name="text">
                <string>Overlaod</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="intermodulation_check">
               <property name="text">
                <string>Intermodulation</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="desensitization_check">
               <property name="text">
                <string>Desensitization</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="protection_class_box">
            <property name="title">
             <string>Protection Level Classification</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QComboBox" name="radio_dropdown">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="protection_legend_table">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="autoScroll">
                <bool>false</bool>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>120</number>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>120</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderMinimumSectionSize">
                <number>30</number>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>Damage</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Overload</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Intermodulation</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Desensitization</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>Protection Level (dBm)</string>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string>30.0</string>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>166</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>-4.0</string>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>99</green>
                   <blue>97</blue>
                  </color>
                 </brush>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>-30.0</string>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>211</red>
                   <green>89</green>
                   <blue>162</blue>
                  </color>
                 </brush>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
               </item>
               <item row="3" column="0">
                <property name="text">
                 <string>-104.0</string>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>125</red>
                   <green>115</green>
                   <blue>202</blue>
                  </color>
                 </brush>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="radio_specific_levels">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Use radio specific protection levels</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="protection_matrix">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="protection_results_btn">
            <property name="text">
             <string>Generate Results</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="protection_export_btn">
            <property name="text">
             <string>Export to Excel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="protection_save_img_btn">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Save Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="interference_tab">
       <attribute name="title">
        <string>Interference Type</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QGroupBox" name="interference_select_box">
            <property name="title">
             <string>Interference Type (Source / Victim)</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QCheckBox" name="in_in_check">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Inband / Inband</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="out_in_check">
               <property name="text">
                <string>Out of Band / Inband</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="in_out_check">
               <property name="text">
                <string>Inband / Out of Band</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="out_out_check">
               <property name="text">
                <string>Out of Band / Out of Band</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="interference_class_box">
            <property name="title">
             <string>Interference Type Classification</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QTableWidget" name="interference_legend_table">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>190</number>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>true</bool>
               </attribute>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>Interference Type (Source / Victim)</string>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string>Inband / Inband</string>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>166</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>Out of Band / Inband</string>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>99</green>
                   <blue>97</blue>
                  </color>
                 </brush>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>Inband / Out of Band</string>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>211</red>
                   <green>89</green>
                   <blue>162</blue>
                  </color>
                 </brush>
                </property>
               </item>
               <item row="3" column="0">
                <property name="text">
                 <string>Out of Band / Out of Band</string>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>125</red>
                   <green>115</green>
                   <blue>202</blue>
                  </color>
                 </brush>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="interference_matrix"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="interference_results_btn">
            <property name="text">
             <string>Generate Results</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="interference_export_btn">
            <property name="text">
             <string>Export to Excel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="interference_save_img_btn">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Save Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
